frontend/src/scenes/PreflightCheck/PreflightCheck.scss

```
@@ -0,0 +1,176 @@
+@import '~/vars';
+
+.preflight-check-container {
+    min-height: 100vh;
+    align-items: center;
+    justify-content: center;
+    display: flex;
+    flex-direction: row;
+    padding: 40px 16px;
+
+    .preflight-box {
+        @extend %mixin-elevated;
```

`%mixin-elevated` is pretty old and not the style we use now (probably should refactor that out). `--shadow-elevation` is the correct shadow.
```suggestion
        box-shadow: var(--shadow-elevation);
```

---

Makes sense, just refactored it out here: https://github.com/PostHog/posthog/pull/9644